code/modules/mod/mod_actions.dm

```diff
@@ -2,29 +2,28 @@
 	background_icon_state = "bg_tech_blue"
 	icon_icon = 'icons/mob/actions/actions_mod.dmi'
 	check_flags = AB_CHECK_CONSCIOUS
+	var/ai_action = FALSE
```

Comment:
Adding some documentation would be very nice for people that want to contribute in the future, and it'd be following the autodoc contributing guidelines ( https://github.com/tgstation/tgstation/blob/master/.github/guides/AUTODOC.md?plain=1#L30 )